addons/website_sale/__manifest__.py

```diff
@@ -7,7 +7,7 @@
     'sequence': 50,
     'summary': 'Sell your products online',
     'website': 'https://www.odoo.com/page/e-commerce',
-    'version': '1.1',
+    'version': '1.2',
```

Comment:
is this because of the new view?

Reply:
Should not be necessary for that, I think I removed a view at one point but then might have reverted it.